.github/workflows/docker.yml

```diff
@@ -72,7 +79,7 @@ jobs:
         env:
           PECAN_GIT_CHECKSUM: ${{ github.sha }}
           PECAN_GIT_BRANCH: ${GITHUB_BRANCH}
-          VERSION: ${PECAN_VERSION}
+          VERSION: ${{ env.PECAN_VERSION }}
```

Comment:
one option is for us take the docker.sh file and make multiple jobs (each bullet can be a matrix)
- depends
- base web docs
- models executor data thredds monitor rstudio-nginx
- dbsync
- all models
-

Reply:
@robkooper Can you say more about why? I'm guessing it's either:
a) If the build run as separate jobs, do they each get their own rate limit and let us avoid bothering to pass the PAT around?

Reply:
I'm partlly hoping that we might get around the dependencies issue, also it should speed up the build process since we can run things in parallel.

The depends image can be build only if the Dockerfile or pecan.depends.R file changes. I hope that we can also avoid running updates to packages, since that should have been captured in the pecan.depends.R file.

Reply:
@robkooper Here's a VERY off-the-cuff first stab at a matrix implementation, branched off this PR for experimenting: https://github.com/PecanProject/pecan/pull/3085

Will definitely fail at first because it doesn't handle order dependencies yet -- feel free to push your improvements straight to that branch.

Reply:
...Actually I think #3805 is backwards: You said "each bullet can be a matrix", I made each bullet into an entry in one single matrix, so it dies on order dependencies.